packages/eos-components/src/components/Keywords.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <span>
+    <b-button
+      v-for="w in words"
+      :key="w"
+      pill
+      variant="outline-primary"
+      class="mx-1 pr-1 py-1"
+      @click="$emit('click', w)"
+    >
+      {{ w }}
+      <slot>
+        <CloseIcon :size="20" class="ml-2" />
```

Comment:
The `size` here is a constant, so I think that it doesn't need to be dynamic. I. e. it doesn't need to be a v-bind (or in this case the `:` shorthand)

Reply:
The `:` is needed because the size on material icon components should be a number and not an string:
```
Invalid prop: type check failed for prop "size". Expected Number with value 20, got String with value "20".
```

In any case, I'm moving this to the `constants.js` file so it's not defined here.

Reply:
Ah OK, good!